fe/fe-core/src/main/java/com/starrocks/persist/DropPartitionInfo.java

```diff
@@ -43,29 +43,30 @@
 import java.io.DataInput;
 import java.io.DataOutput;
 import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
 
 public class DropPartitionInfo implements Writable {
     @SerializedName(value = "dbId")
     private Long dbId;
     @SerializedName(value = "tableId")
     private Long tableId;
-    @SerializedName(value = "partitionName")
-    private String partitionName;
```

Comment:
You cannot delete the partitionName attribute. If you delete it, when reading the old format DropPartitionInfo, partitionNames will be empty, causing replayDropPartition to fail.
If you want to use `List<String> partitionNames` to store the deleted partitions, you should add the `String partitionName`(if not null) to `partitionNames` in the gsonPostProcess hook, please refer to https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

![image](https://github.com/StarRocks/starrocks/assets/54172532/0a1913c5-008e-4f23-bd15-42dcff53dd2a)

![image](https://github.com/StarRocks/starrocks/assets/54172532/32624533-acd0-46c3-ab21-c66909893816)

Reply:
Thank you for gengjun's code review. I have fixed these issues.